### ISSUE TITLE
Training Pong with DQN

### ISSUE THREAD

**Issue by mklissa** [matthiasplappert/keras-rl#76]
Hi there,
awesome repository by the way. I tried to train the DQN on the game of Pong but the results seem to be pretty bad:
Testing for 10 episodes ...
Episode 1: reward: -21.000, steps: 764
Episode 2: reward: -20.000, steps: 844
Episode 3: reward: -21.000, steps: 764
Episode 4: reward: -21.000, steps: 764
Episode 5: reward: -21.000, steps: 764
Episode 6: reward: -21.000, steps: 764
Episode 7: reward: -21.000, steps: 764
Episode 8: reward: -21.000, steps: 764
Episode 9: reward: -21.000, steps: 820
Episode 10: reward: -21.000, steps: 764

This is after 1.2M steps. It this normal?

Thanks

**Comment by matthiasplappert** [comment]
Are you using the TensorFlow backend for Keras?

**Comment by mklissa** [comment]
Nope, I am using the Theano backend. It works well with Breakout though, however with Pong I couldn't get any results.

**Comment by matthiasplappert** [comment]
Could be that you didn't train long enough, I did have some success with Pong in the past. Also, please post questions like this on the keras-rl Google group: https://groups.google.com/forum/#!forum/keras-rl-users

**Comment by hypnopump** [comment]
Same issue here. not solved
